internal/db/migrations/2025-08-13.0-drop_ingest_payments_table.sql

```diff
@@ -0,0 +1,26 @@
+-- +migrate Up
```

Comment:
Normally, when removing prior tables, we would introduce a drop migration but in this case, since we have not officially released WB, I was thinking of just deleting the migration files for unused tables. This way we release wallet backend with a proper migration with only the tables that are used. What are your thoughts? @JakeUrban @akcays

Reply:
Yea I'm ok with that approach. We should reach out to known users of the wallet backend prior to merging to main though so they have the opportunity to fork the repo if they want to keep their own applications stable.